PEAK_MIMO.py

```diff
@@ -31,90 +31,109 @@ def PEAK_MIMO(w_start, w_end, error_poles_direction, wr, deadtime_if=0):
 
             Qzp = yz_direction.H*yp_direction/(yzp_mat1-yzp_mat2)
 
-            if deadtime_if==0:
-                #this matrix is the matrix from which the SVD is going to be done to determine the final minimum peak
-                pre_mat = (sc_lin.sqrtm((np.linalg.inv(Qz)))*Qzp*(sc_lin.sqrtm(np.linalg.inv(Qp))))
+            if deadtime_if == 0:
+                # This matrix is the matrix from which the SVD is going to
+                # be done to determine the final minimum peak
+                pre_mat = (sc_lin.sqrtm((np.linalg.inv(Qz))) * 
+                           Qzp*(sc_lin.sqrtm(np.linalg.inv(Qp))))
 
-                #final calculation for the peak value
+                # Final calculation for the peak value
                 Ms_min = np.sqrt(1+(np.max(np.linalg.svd(pre_mat)[1]))**2)
                 print('')
                 print('Minimum peak values on T and S without deadtime')
                 print('Ms_min = Mt_min = ', Ms_min)
                 print('')
 
-            #Skogestad eq 6-16 pg 226 using maximum deadtime per output channel to give tightest lowest bounds
+            # Skogestad eq 6-16 pg 226 using maximum deadtime per output
+            # channel to give tightest lowest bounds
             if deadtime_if == 1:
-                #create vector to be used for the diagonal deadtime matrix containing each outputs' maximum dead time
-                #this would ensure tighter bounds on T and S
-                #the minimum function is used because all stable systems have dead time with a negative sign
+                # Create vector to be used for the diagonal deadtime matrix
+                # containing each outputs' maximum dead time
+                # this would ensure tighter bounds on T and S
+                # the minimum function is used because all stable systems
+                # have dead time with a negative sign
 
                 dead_time_vec_max_row = np.zeros(deadtime()[0].shape[0])
 
                 for i in range(deadtime()[0].shape[0]):
                     dead_time_vec_max_row[i] = np.max(deadtime()[0][i, :])
 
-
                 def Dead_time_matrix(s, dead_time_vec_max_row):
 
-                    dead_time_matrix = np.diag(np.exp(np.multiply(dead_time_vec_max_row, s)))
+                    dead_time_matrix = np.diag(
+                        np.exp(np.multiply(dead_time_vec_max_row, s)))
                     return dead_time_matrix
 
                 Q_dead = np.zeros([G(0.0001).shape[0], G(0.0001).shape[0]])
 
                 for i in range(len(Poles_G)):
                     for j in range(len(Poles_G)):
-                        denominator_mat= np.transpose(np.conjugate(yp_direction[:, i]))*Dead_time_matrix(Poles_G[i], dead_time_vec_max_row)*Dead_time_matrix(Poles_G[j], dead_time_vec_max_row)*yp_direction[:, j]
+                        denominator_mat = (
+                           np.transpose(np.conjugate(yp_direction[:, i]))*
```

Comment:
I changed this line so that the operator is after the break. I read this on the PEP8 documentation but I do not think this has been incorporated into Code Climate yet. Because when I used this convention in other code, Code Climate still reported an issue. If my change does not show up as an issue in Code Climate I will change the other lines as recommended. Thanks for the review and recommendations.